src/components/NanoContract/NanoContractDetails.component.js

```diff
@@ -43,7 +43,7 @@ const getNanoContractDetails = (ncId) => (state) => {
    * quickly, therefore the user will not have time to see the default state.
    */
   const defaultMeta = { isLoading: false, error: null, after: null };
-  const txHistory = Object.values(state.nanoContract.history[ncId] || {});
+  const txHistory = Object.values(state.nanoContract.history[ncId] || []);
```

Comment:
I think we discussed it last week, but you were going to remove this `Object.values`, right?

Reply:
Yes, I removed it.